src/plot_api/plot_schema.js

```diff
@@ -56,16 +56,17 @@ PlotSchema.get = function() {
     return plotSchema;
 };
 
-PlotSchema.crawl = function(attrs, callback) {
+PlotSchema.crawl = function (attrs, callback, specifiedLevel) {
+    var level = specifiedLevel || 0;
```

Comment:
True, the crawler is already _accessible_ publicly on `Plotly.PlotSchema.crawl` though it isn't an _official_ plotly.js method yet.

Maybe we should add it the main `Plotly` object in `v2.0.0` e.g. `Plotly.crawlAttributes` ?